src/dependencies/dss-flappers/FlapperInit.sol

```diff
@@ -0,0 +1,109 @@
+    function daiJoin() external view returns (address);
+    function spotter() external view returns (address);
+    function pip() external view returns (address);
+    function pair() external view returns (address);
+    function gem() external view returns (address);
+    function rely(address) external;
+    function file(bytes32, uint256) external;
+    function file(bytes32, address) external;
+}
+
+interface FlapperMomLike {
+    function setAuthority(address) external;
+}
+
+interface OracleWrapperLike {
+    function pip() external view returns (address);
+}
+
+interface PipLike {
+    function kiss(address) external;
+}
+
+interface PairLike {
+    function token0() external view returns (address);
+    function token1() external view returns (address);
+}
+
+struct FlapperUniV2Config {
+    uint256 hop;
+    uint256 want;
+    address pip;
+    uint256 hump;
+    uint256 bump;
+}
+
+library FlapperInit {
+    uint256 constant WAD = 10 ** 18;
+
+    function initFlapperUniV2(
+        DssInstance        memory dss,
+        FlapperInstance    memory flapperInstance,
+        FlapperUniV2Config memory cfg
+    ) internal {
+        FlapperUniV2Like flapper = FlapperUniV2Like(flapperInstance.flapper);
+        FlapperMomLike   mom     = FlapperMomLike(flapperInstance.mom);
+
+        // Sanity checks
+        require(flapper.vat()     == address(dss.vat),     "Flapper vat mismatch");
+        require(flapper.daiJoin() == address(dss.daiJoin), "Flapper daiJoin mismatch");
+        require(flapper.spotter() == address(dss.spotter), "Flapper spotter mismatch");
+
+        PairLike pair = PairLike(flapper.pair());
+        (address pairDai, address pairGem) = pair.token0() == address(dss.dai) ? (pair.token0(), pair.token1())
+                                                                               : (pair.token1(), pair.token0());
+        require(pairDai == address(dss.dai), "Dai mismatch");
+        require(pairGem == flapper.gem(),    "Gem mismatch");
+
+        require(cfg.hop >= 5 minutes, "hop too low");
+        require(cfg.want >= WAD * 90 / 100, "want too low");
+        require(cfg.hump > 0, "hump too low");
+
+        flapper.file("hop",  cfg.hop);
+        flapper.file("want", cfg.want);
+        flapper.file("pip",  cfg.pip);
+        flapper.rely(address(dss.vow));
+        flapper.rely(address(mom));
+
+        dss.vow.file("flapper", address(flapper));
+        dss.vow.file("hump",    cfg.hump);
+        dss.vow.file("bump",    cfg.bump);
+
+        mom.setAuthority(dss.chainlog.getAddress("MCD_ADM"));
+
+        dss.chainlog.setAddress("MCD_FLAP",    address(flapper));
+        dss.chainlog.setAddress("FLAPPER_MOM", address(mom));
+    }
+
+    function initDirectOracle(address flapper) internal {
+        PipLike(FlapperUniV2Like(flapper).pip()).kiss(flapper);
+    }
+
+    function initOracleWrapper(DssInstance memory dss, address wrapper, bytes32 clKey) internal {
```

Comment:
Is this unused? So the [wrapper](https://github.com/makerdao/dss-flappers/blob/6a8d6bedec27bc871a1cf1caad59014b3c0e2937/src/OracleWrapper.sol) is not used?

Reply:
Right.

Reply:
Do you want to remove it then or what's the point of deploying it?

Reply:
The same of copying the exact same deployment files than were audited.